CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 * Add `changeEvent` to `Attribute`.
+* [Polymer] Extract 'listeners' from 1.0-style declarations.
```

Comment:
Minor, I can do this when it's merged, but if you get the chance to rebase, this will need to move up to the `## Unreleased` section again, as we stamped a new release last night.